pulumi/infra/nomad_server_cluster.py

```diff
@@ -0,0 +1,171 @@
+import json
+from typing import Optional, Sequence
+
+import pulumi_aws as aws
+from infra import policies
+from infra.ami_artifacts import get_ami_id
+from infra.config import DEPLOYMENT_NAME, DGRAPH_LOG_RETENTION_DAYS
+from infra.ec2 import Ec2Port
+from infra.ec2_cluster import Ec2Cluster
+from infra.network import Network
+from infra.policies import EC2_DESCRIBE_INSTANCES_POLICY
+
+import pulumi
+
+
+class NomadServer(pulumi.ComponentResource):
```

Comment:
Isn't this actually a `NomadCluster` because it's spread across 3 machines?

Reply:
yep, done